docs_user/modules/proc_adopting-the-compute-service.adoc

```diff
@@ -2,44 +2,29 @@
 
 = Adopting the {compute_service}
 
+To adopt the {compute_service_first_ref}, you patch an existing `OpenStackControlPlane` custom resource (CR) where the {compute_service} is disabled. The patch starts the service with the configuration parameters that are provided by the {rhos_prev_long} ({OpenStackShort}) environment.
+
 [NOTE]
-This example scenario describes a simple single-cell setup. Real
-multi-stack topology recommended for production use results in different
-cells DBs layout, and should be using different naming schemes (not covered
-here this time).
+The following example scenario describes a single-cell setup. Real
+multi-stack topology that is recommended for production use results in cells having a different database layout, and should use different naming schemes.
 
 .Prerequisites
 
-* Previous Adoption steps completed. Notably,
- ** the xref:migrating-databases-to-mariadb-instances_migrating-databases[Migrating databases to MariaDB instances]
-must already be imported into the control plane MariaDB;
- ** the xref:adopting-the-identity-service_adopt-control-plane[Adopting the Identity service] needs to be imported;
- ** the xref:adopting-the-key-manager-service_adopt-control-plane[Adopting the Key Manager service] needs to be imported;
- ** the xref:adopting-the-placement-service_{context}[Adopting the Placement service] needs to be imported;
- ** the xref:adopting-the-image-service_adopt-control-plane[Adopting the Image service] needs to be imported;
- ** the xref:migrating-ovn-data_migrating-databases[Migrating OVN data] need to be imported;
- ** the xref:adopting-the-networking-service_adopt-control-plane[Adopting the Networking service] needs to be imported;
-** the {bare_metal} needs to be imported;
-//kgilliga:I removed the link because it did not work. I might rewrite this section anyway.
- ** Required topology-specific service configuration. For more information, see
-xref:proc_retrieving-topology-specific-service-configuration_migrating-databases[Retrieving topology-specific service configuration].
- ** {rhos_prev_long} services have been stopped. For more information, see xref:stopping-openstack-services_migrating-databases[Stopping {rhos_prev_long} services].
-* Define the following shell variables. The values that are used are examples. Replace these example values with values that are correct for your environment:
```

Comment:
@SeanMooney FYI: I removed most of the prerequisites because the customer should be following the guide in order, and they would have done all of this already. But if there’s a prerequisite that you think we should emphasize, please let me know.

Reply:
I would prefer retaining the list of dependencies explicit

Reply:
**Note to peer reviewers**: I'd like a second opinion on listing prerequisites. If customers should be following the guide in order, do we need to list every procedure that they already should have done?

Reply:
You could leave in just the first prerequisite to be explicit. 
Previous Adoption steps completed. 
Reword:
You have completed the previous Adoption steps.